indico/modules/events/editing/controllers/frontend.py

```diff
@@ -51,10 +51,19 @@ def _process(self):
 
 
 class RHManageEditingFileTypes(RHEditingManagementBase):
+    def _process_args(self):
+        RHEditingManagementBase._process_args(self)
+        self.editable_type = request.view_args['type']
```

Comment:
I think it would be more consistent to use `EditableType[request.view_args['type']]` here and below, and then use the `.name` of the enum member when passing it to the template (or inside the template).